ext/rotoscope/rotoscope.c

```diff
@@ -180,6 +180,7 @@ static void event_hook(VALUE tpval, void *data)
   if (in_fork(config))
   {
     rb_tracepoint_disable(config->tracepoint);
+    config->state = RS_OPEN;
```

Comment:
if we're inside of a fork, I'm thinking we should set the state to `RS_CLOSED` since we want to block any writing into the file. In rubyland you can ask `rotoscope.closed?` and that should determine if it's valid to `trace`. Thoughts?